src/Toggler.js

```diff
@@ -44,9 +44,11 @@ class Toggler extends State {
 			return content;
 		}
 
-		content = header.querySelector(this.content);
-		if (content) {
-			return content;
+		if (header.querySelector) {
```

Comment:
Hey, @NolanChan. I was just wondering on this validation. Could you explain why are you checking if `header` has the `querySelector` method?